packages/gelato-adapter/README.md

```diff
@@ -12,11 +12,43 @@ Also make sure to install the peer dependencies required by the adapter if your
 yarn add ethers viem @reservoir0x/reservoir-sdk
 ```
 
-### Usage
+## Usage
 
-To use the adapter pass in your ethers signer and gelato api key. In return receive a normalized ReservoirWallet object:
+There are two ways to use this adapter: with a Gelato API key or a Gelato API proxy URL.
 
+If you are relaying transactions for a public protocol like Seaport, you might prefer to use the Gelato SDK on the backend to not expose your API keys as these will give anyone the power to relay their transactions. In this case, you should use the Gelato API proxy URL parameter. You will have to build your own backend API proxy, there is a basic example [here](/sdk-demo/app/api/relay/route.ts).
+
+Otherwise, if you are relaying transactions and don't mind your API keys being public, you can pass the Gelato API keys into the adapter directly on the front-end.
+
+To use the adapter pass in your ethers signer and Gelato API key / Gelato API proxy URL. In return receive a normalized ReservoirWallet object:
+
+### Gelato API key
+<h4>Sequence Diagram</h4>
+
+```mermaid
+sequenceDiagram
```

Comment:
I've never used this, pretty neat. Is there any way for us to change the purple to something that works better when the text overlays it? Or change the text to something more neutral (gray).

Reply:
Yeah, will test in light and dark mode 😆